### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by 3scava1i3r**
Added terminology section inside README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,23 @@
...
 HTTP Proxy Caching for Kong
 
+
 ## Synopsis
 
 This plugin provides a reverse proxy cache implementation for Kong. It caches response entities based on configurable response code and content type, as well as request method. It can cache per-Consumer or per-API. Cache entities are stored for a configurable period of time, after which subsequent requests to the same resource will re-fetch and re-store the resource. Cache entities can also be forcefully purged via the Admin API prior to their expiration time.
 
+## Terminology
+
+- `plugin`: a plugin executing actions inside Kong before or after a request has been proxied to the upstream API.
```
Comment (kikito) [inline README.md:14]:
```suggestion
- `Plugin`: a plugin executing actions inside Kong before or after a request has been proxied to the upstream API.
```

File: README.md [hunk 2]
```diff
@@ -4,10 +4,23 @@
...
+- `Service`: the Kong entity representing an external upstream API or microservice.
+- `Route`: the Kong entity representing a way to map downstream requests to upstream services.
+- `Consumer`: an entity that makes requests for Kong to proxy; it represents either a user or an external service.
+- `Credential`: a unique string associated with a Consumer, also referred to as an API key.
+- `Upstream service`: this refers to your own API/service sitting behind Kong, to which client requests are forwarded.
```
Comment (kikito) [inline README.md:19]:
```suggestion
- `Upstream`: this refers to your own API/service sitting behind Kong, to which client requests are forwarded.
```

**Comment by 3scava1i3r** [comment]
How is it that after editing the README.md file our test are getting failed?

**Comment by hishamhm** [comment]
This text seems to be copied over from an older version https://docs.konghq.com/hub/kong-inc/proxy-cache/ — should we instead just add a link to the official documentation in the README, rather than duplicate it here?

**Comment by hishamhm** [comment]
This text seems to be copied over from an older version of https://docs.konghq.com/hub/kong-inc/proxy-cache/ — I think we should instead just add a link to the official documentation in the README, rather than duplicate it here. I'll go ahead and make the switch.

**Pull request closed without merging**